src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -862,8 +866,12 @@ public boolean isSelectQuestionNumberEnabled(int qnNumber) {
     }
 
     public int getSelectedQuestionNumber(int qnNumber) {
-        Select qnNumSelect = new Select(getSelectQuestionNumberDropdown(qnNumber));
-        return Integer.parseInt(qnNumSelect.getFirstSelectedOption().getText().trim());
+        if (isQuestionEnabled(qnNumber)) {
+            Select qnNumSelect = new Select(getSelectQuestionNumberDropdown(qnNumber));
+            return Integer.parseInt(qnNumSelect.getFirstSelectedOption().getText().trim());
+        }
+        String questionNumber = getSelectQuestionNumberStatic(qnNumber).getText();
+        return Integer.parseInt(questionNumber.substring(0, questionNumber.length() - 1));
```

Comment:
`.substring(startIndex, endIndex)`: startIndex is inclusive and endIndex is exclusive. So what's the rationale of doing `questionNumber.length() - 1` ?

On a side note, does `return Integer.parseInt(getSelectQuestionNumberStatic(qnNumber).getText())` not work?

Reply:
getSelectQuestionNumberStatic(qnNumber) outputs ${fqForm.questionIndex}: so the point of trimming is to remove the colon. 
`return Integer.parseInt(getSelectQuestionNumberStatic(qnNumber).getText())` would throw a number format exception for that reason.

Reply:
I see. You can add a comment here describing what you mentioned as it is not easy to spot on first sight.